test/Microsoft.DotNet.ApiSymbolExtensions.Tests/AssemblySymbolLoaderTests.cs

```diff
@@ -6,12 +6,15 @@
 using System.Collections.Concurrent;
 using System.Reflection;
 using Microsoft.CodeAnalysis;
+using Microsoft.DotNet.ApiCompatibility.Tests;
```

Comment:
The ApiSymbolExtensions library nor the tests should depend on APICompat or the ApiCompatibility library. That's the wrong layer.